DB/projects.json

```
@@ -48,7 +48,7 @@
         "description": "FileDrop is an FileSharing Application ",
         "tech": [
           "reactjs",
-          "Nodejs",
+          "nodejs",
           "jwt",
           "js",
```

change to `javascript`
```suggestion
          "javascript",
```